aea/protocols/generator/common.py

```diff
@@ -364,14 +364,13 @@ def check_prerequisites() -> None:
             "Cannot find protocol buffer compiler! To install, please follow this link: https://developers.google.com/protocol-buffers/"
         )
 
+def get_protoc_version() -> str:
+    """Get the protoc version used."""
     result = subprocess.run(  # nosec
         ["protoc", "--version"], stdout=subprocess.PIPE, check=True
     )
     result_str = result.stdout.decode("utf-8")
-    if LIBPROTOC_VERSION not in result_str:
-        raise FileNotFoundError(  # pragma: nocover
-            f"Invalid version for protoc. Found: {result_str}. Required: {LIBPROTOC_VERSION}."
-        )
```

Comment:
We could keep a minimum version check, but I think its not needed as the protobuf schema will enforce that.